scripts/flowkiller.py

```diff
@@ -36,6 +39,14 @@ class FlowKiller(Application):
         """,
     )
 
+    banned_ipv4_file_glob = Unicode(
```

Comment:
Can we turn this into a list of globs instead? We may have to do that in the future, and I think it's easier to start off that way.

You can merge this after this change if that feels right to you. THANK YOU SO MUCH!

Reply:
Done, thanks!